spec/models/sent_scheduled_message_spec.rb

```diff
@@ -35,4 +35,15 @@
   describe "Delegated methods" do
     it { should delegate_method(:slack_username).to(:employee) }
   end
+
+  describe "Scopes" do
+    describe "self.by_year" do 
```

Comment:
very nice! this is super small but I usually put `.by_year` in the description for class methods (if it were an instance method, I'd do `#by_year`)